spec/unit/helpers/check_helper_spec.rb

```diff
@@ -48,4 +48,27 @@
     params.setup_filters(PARAMS_3)
     expect { params[:test_param_three] }.to raise_error(/not defined in the specification/)
   end
+
+  context 'when created from stringified hash' do
+    let(:raw_params) do
+      { 'update_preference' => { 'keep_data' => false,
+                                 'allow_cascade_drops' => true } }
+    end
+
+    let(:params) do
+      GoodData::LCM2.convert_to_smart_hash(raw_params)
+    end
+
+    let(:spec) do
+      GoodData::LCM2::BaseAction.define_params(self) do
```

Comment:
Similar code found in check_helper_spec.rb:40, check_helper_spec.rb:63 (mass = 38)